ui/app/AppLayouts/Chat/views/UserListsView.qml

```diff
@@ -0,0 +1,176 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import shared 1.0
+import shared.panels 1.0
+import shared.status 1.0
+import shared.views.chat 1.0
+import utils 1.0
+
+import "../panels"
+
+import AppLayouts.Chat.stores 1.0 as ChatStores
+
+Item {
+    id: root
+
+    property ChatStores.RootStore store
+    property var chatCommunitySectionModule
```

Comment:
module object should not be used directly, the whole access to the backend should be hidden behind stores (as explained in section regarding stores in the guide).

`chatCommunitySectionModule` comes from `ChatStores.RootStore` and only that store should be taken as a dependency here. The store itself should expose everything you need from `chatCommunitySectionModule`.
Most of methods is already exposed they should be, e.g.:
```
    function getMySectionId() {
        return chatCommunitySectionModule.getMySectionId()
    }
```
If sth is missing, please just add it.